radio/src/pulses/modules_helpers.h

```diff
@@ -543,6 +573,28 @@ inline void resetAccessAuthenticationCount()
 #endif
 }
 
+inline void resetAfhds3Options(uint8_t moduleIdx){
+  g_model.moduleData[moduleIdx].rfProtocol = 0;
```

Comment:
auto & module = g_model.moduleData[moduleIdx];

Reply:
Fixed as requested - compiler is optimising it anyhow.